src/forms/registration-popup/index.jsx

```diff
@@ -109,20 +108,14 @@ const RegistrationForm = () => {
 
   useEffect(() => {
     if (thirdPartyAuthApiStatus === COMPLETE_STATE) {
-      if (providers.length > 0) {
+      if (providers.length > 0 && socialAuthnButtonRef.current) {
         socialAuthnButtonRef.current.focus();
       } else if (emailRef.current) {
         emailRef.current.focus();
       }
     }
   }, [thirdPartyAuthApiStatus, providers]);
 
-  useEffect(() => {
```

Comment:
Should the focus will not be go to the Alert when there is an error or should it be on the button?

Reply:
Screen Reader will read the error Alert message and focus should be on the button.